code/game/area/Space Station 13 areas.dm

```diff
@@ -1795,14 +1803,6 @@ area/security/podbay
 	name = "Security Post - Science"
 	icon_state = "checkpoint1"
 
-/area/security/vacantoffice
-	name = "\improper Vacant Office"
-	icon_state = "security"
-
-/area/security/vacantoffice2

```

Comment:
tbh may as well leave these in.

Reply:
👍 otherwise

Reply:
eh I'd rather keep it removed so in the future if someone needs to make a vacant office area there isn't confusion about which one to use